bindings/go/evmc/types.go

```diff
@@ -0,0 +1,18 @@
+// EVMC: Ethereum Client-VM Connector API.
+// Copyright 2020 The EVMC Authors.
+// Licensed under the Apache License, Version 2.0.
+package evmc
+
+// Lengths of hashes and addresses in bytes.
+const (
+	// hashLength is the expected length of the hash (in bytes)
+	hashLength = 32
+	// addressLength is the expected length of the address (in bytes)
+	addressLength = 20
+)
+
+// Hash represents the 32 bytes of arbitrary data (e.g. the result of Keccak256 hash).
+type Hash [hashLength]byte
+
+// Address represents the 20 byte address of an Ethereum account.
+type Address [addressLength]byte
```

Comment:
Rookie go question: are the const needed here or could something like `Address.len` work?

Reply:
I think `len(Address)` should work in `const` context in asserts...

Reply:
I’m not sure I understand the question. 
You mean just do `type Address [20]byte` here and use `len(Address)` in the other places? It should work theoretically. Here I mostly repeated how Go Ethereum does it.

Reply:
> You mean just do type Address [20]byte here and use len(Address) in the other places?

Yes I meant that. It seems nicer from my side, but I have no strong preference.

Reply:
I kept it as close as in go-ethereum as possible, and I don't have any strong preferences there either.

Reply:
@chfast you decide.